models/User.model.js

```diff
@@ -4,11 +4,17 @@ const { Schema, model } = require("mongoose");
 const userSchema = new Schema({
   username: {
     type: String,
+    unique: true,
+    trim: true,
+    required: [true, "Username is required."],
     unique: true
   },
-  password: String
+  password: {
+    type: String,
+    required: [true, "Password is required."]
+  }
 });
 
 const User = model("User", userSchema);
 
-module.exports = User;
+module.exports = model("User", userShema);
```

Comment:
- you could have just exported the `User` variable, there is no need to call the model function again.
- careful with typos; here it should be `userSchema`, not `userShema`